packages/react-instantsearch-dom/src/widgets/__tests__/Answers.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import algoliasearch from 'algoliasearch';
+import { InstantSearch, SearchBox } from 'react-instantsearch-dom';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+import MutationObserver from '@sheerun/mutationobserver-shim';
+import Answers from '../Answers';
+
+describe('Answers', () => {
+  let searchClient;
+  beforeEach(() => {
+    searchClient = algoliasearch(
+      'CKOEQ4XGMU',
+      '6560d3886292a5aec86d63b9a2cba447'
```

Comment:
why not test this with a fake search client? that way it can't be flaky

Reply:
Yeah I wanted to make sure if the test cases really work well. So I went with the real appId. Once I confirmed both the code and the test code work well, I wondered if I should replace it with the fake client. Maybe I should, otherwise it may fail.

Reply:
sounds good for testing, but the final test should be a fake client I think

Reply:
yup just updated it https://github.com/algolia/react-instantsearch/pull/2996/commits/3b5f980689bf728e7c114b1aabd90175b4088308